### DISCUSSION TITLE
Does Avalonia.Automation also work for non-desktop apps?

### DISCUSSION THREAD

**Discussion by ChristoWolf** [AvaloniaUI/Avalonia#8513] [Q&A] [answered]
Hi!

All of this was done the master branch, commit 803e6dc9d0a8894169470084cec126c15a705a0c.

When I attach an `AutomationProperties.AutomationId` in an Android app, it seems to not be exposed to the automation tree (`content-desc` does not contain the set value).
Also, it seems that `uiautomationviewer` does not recognize any elements.

Doing the same with a Windows app works as expected, i.e. any inspection tool shows the correct `AutomationId`s and most elements are exposed to the automation tree.

Has anybody else experienced with Avalonia's automation on other platforms?

**Comment by maxkatz6** [comment, marked as answer]
No, it was implemented only for Windows and macOS so far.
We lack experience with android automation, and desktop was in priority so far. Any help will be welcomed.

**Reply by ChristoWolf** [reply]
Thanks for the info!
If I can find the time, I would love to help out, as cross-platform automatability would be awesome.
